src/mongoose-cleaner.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
```

Comment:
We might want to skip lodash altogether, it is fairly trivial to do these two natively.